guide/samples/Cargo.toml

```diff
@@ -28,13 +28,14 @@ categories.workspace = true
 name = "getting_started"
 
 [dependencies]
-crc32c = "0.6"
-tokio  = { version = "1", features = ["full", "macros"] }
+crc32c  = "0.6"
+tokio   = { version = "1", features = ["full", "macros"] }
+futures = { version = "0.3" }
 # ANCHOR: longrunning
 google-cloud-longrunning = { version = "0.24", path = "../../src/generated/longrunning" }
 # ANCHOR_END: longrunning
 # ANCHOR: gax
-google-cloud-gax = { version = "0.22", path = "../../src/gax" }
+google-cloud-gax = { version = "0.22", features = ["unstable-stream"], path = "../../src/gax" }
 # ANCHOR_END: gax
```

Comment:
comment: this anchor is used elsewhere, and the `features = ["unstable-stream"]` bit will not be relevant.

I don't have a good solution, but just pointing this out.

Reply:
We can take inspiration from this:

https://github.com/googleapis/google-cloud-rust/blob/97de3736af82082d0ab2711ad71123ab425b1459/guide/samples/Cargo.toml#L51-L57

And do like:

```toml
Lines 51 to 57 in [97de373](https://github.com/googleapis/google-cloud-rust/commit/97de3736af82082d0ab2711ad71123ab425b1459)

 # ANCHOR: gax 
 # ANCHOR: gax-with-streams
 [dependencies.google-cloud-gax] 
 version  = "0.22" 
 # ANCHOR_END: gax 
 features = ["unstable-stream"] 
 # ANCHOR_END: gax-with-streams
 path = "../../src/gax"
```